backend/organization/models/members.py

```diff
@@ -114,3 +114,76 @@ class Meta:
 
     def __str__(self):
         return "User %d member for organization %s" % (self.user.id, self.organization.name)
+
+class MembershipRequests(models.Model):
+
+    id = models.AutoField(help_text='identifier of a join request'
+                                , primary_key=True
+                                ,verbose_name="join request Id")
+
+
+    user = models.ForeignKey(
+        User,
+        verbose_name="feedback_user",
+        help_text="Points to the user who sent the request",
+        on_delete=models.CASCADE,
+        null=False,
+        related_name='users'
```

Comment:
`related_name` is too broad here. You can either not set a related_name (let django handle it) or set something like `user_membership_request`. A bit more self explanatory related  name.

Reply:
For my own understanding, can I just update this line to `user_membership_request`? Or would I have to propagate that change elsewhere too?